### ISSUE TITLE
Add Images on board.

### ISSUE THREAD

**Issue by goliakshay357** [Canvasbird/canvasboard#85]
Adding Images on board which are can be draggable.

path: /board
Component: board

**Comment by SkullCarverCoder** [comment]
Hi I would love to contribute ! Really interested in this project. Images would be able to be resized? Have an specified file size limit given a configuration parameter/variable? Anyways I would love to work on this as soon as October starts.

**Comment by goliakshay357** [comment]
@SkullCarverCoder sure :+1: 
Yes Images can be resized and there is nothing specific to file limit.
Thanks!

**Comment by goliakshay357** [comment]
Hi @SkullCarverCoder,

Are you working on this issue?

**Comment by SkullCarverCoder** [comment]
Yes ! I am , sorry for taking so long I would be making the pull request associated with this on the weekend , have been tweaking some stuff between working hours 😅but on the weekend I should finish and test everything.

**Comment by goliakshay357** [comment]
Hey @SkullCarverCoder, 
Not at all a problem. You can always contribute to the open issues at your time of convenience :smiley: 

Thanks
